etc/test_base.out

```diff
@@ -3,6 +3,15 @@ testing/test_base
 RADIUS request for 00:aa:bb:cc:dd:ee
 auth_callback for 00:aa:bb:cc:dd:ee: access:Authorized segment:test role:test
 0
+Testing connectivity
```

Comment:
Why is this included twice (in test_base.out and test_flat.out)?

Reply:
This was used to run the test_flat independently. Removed now..